src/ramses_tx/parsers.py

```diff
@@ -810,7 +810,7 @@ def parser_0418(payload: str, msg: Message) -> PayDictT._0418 | PayDictT._0418_N
     return full_result
 
 
-# unknown_042f, from STA, VMS
+# unknown_042f, from STA, VMS remotes coupling
```

Comment:
The `042F` packet payload is not well understood.

Do you have any further information? What do you mean by "remotes coupling"

Do you have any packets?

Reply:
I saw several 042F messages when (un)pairing the Vasco CO2 remote. Here's the complete sequence. 32:022222 is the Vasco D60 HRU unit that it's paired to.

```
CO2 (VMS-17C01) 37:117647

disconnect all
10 sec on touch bottom right:
2024-10-09T11:50:24.287879 065 RQ --- 37:117647 32:022222 --:------ 22F1 001 00
2024-10-09T11:50:25.304920 063 RQ --- 37:117647 32:022222 --:------ 22F1 001 00
2024-10-09T11:50:26.326969 060 RQ --- 37:117647 32:022222 --:------ 22F1 001 00
2024-10-09T11:50:37.839476 068  I --- 37:117647 --:------ 37:117647 042F 009 00012521250026C870
2024-10-09T11:50:40.936596 071 RQ --- 37:117647 32:022222 --:------ 22F1 001 00
2024-10-09T11:50:41.963654 070 RQ --- 37:117647 32:022222 --:------ 22F1 001 00
2024-10-09T11:50:54.902520 064  W --- 37:117647 32:022222 --:------ 1FCA 009 0001FF95CB8FFFFFFF
2024-10-09T11:50:55.918057 067  W --- 37:117647 32:022222 --:------ 1FCA 009 0001FF95CB8FFFFFFF
2024-10-09T11:50:57.755704 061  I --- 37:117647 --:------ 37:117647 042F 009 00022521250027F080
2024-10-09T11:51:49.472184 059  I --- 37:117647 63:262142 --:------ 10E0 038 000001C822060166FEFFFFFFFFFF160C07E0564D532D31374330310000000000000000000000

connect
3 sec on touch bottom right (D60 off):
2024-10-09T10:33:53.714590 057  I --- 37:117647 32:022222 --:------ 31E0 004 00000000
2024-10-09T10:34:50.432196 067  I --- 37:117647 --:------ 37:117647 042F 009 00012420240025C870
2024-10-09T10:34:54.556484 065 RQ --- 37:117647 32:022222 --:------ 22F1 001 00
2024-10-09T10:34:55.573584 063 RQ --- 37:117647 32:022222 --:------ 22F1 001 00
2024-10-09T10:35:02.278347 066  I --- 37:117647 63:262142 --:------ 1FC9 030 0031E095CB8F00129895CB8F0022F195CB8F6610E095CB8F001FC995CB8F
2024-10-09T10:35:04.342107 060  I --- 37:117647 63:262142 --:------ 1FC9 030 0031E095CB8F00129895CB8F0022F195CB8F6610E095CB8F001FC995CB8F
2024-10-09T10:35:09.245476 060 RQ --- 37:117647 32:022222 --:------ 22F1 001 00
2024-10-09T10:35:10.257671 058 RQ --- 37:117647 32:022222 --:------ 22F1 001 00
2024-10-09T10:35:11.269793 056 RQ --- 37:117647 32:022222 --:------ 22F1 001 00
```

Reply:
This PR is about decoding `042F` - my impression is that there is not enough here to work with to achieve that goal.

I suggest we close this PR, but you can certainly have a play and let me know if you have any insights how to progress.